services/QuillLMS/app/views/pages/connect_tool.erb

```diff
@@ -22,7 +22,7 @@
     </section>
 
     <section class="tool-animation-section " id="overview">
-          <video class='tool-animation lazyload' data-src=<%= "#{ENV['CDN_URL']}/videos/tools/connect.mp4" %> autoplay loop/>
+          <video class='tool-animation lazyload' data-src=<%= "#{ENV['CDN_URL']}/videos/diagnostic.mp4" => autoplay loop/>
```

Comment:
Is this supposed to be the diagnostic video?

Reply:
good catch, copy pasta